en/node-operators-guide/por.md

```diff
@@ -0,0 +1,86 @@
+# Orakl Network Proof of Reserve
+
+## Description
+
+The **Orakl Network Proof of Reserve** is a critical component within the Orakl Network ecosystem, designed to establish and verify the reserve holdings of financial entities. This service ensures transparency and trust in the financial operations by providing a secure and auditable process for validating reserve data. The Orakl Network Proof of Reserve is a cornerstone within the ecosystem, dedicated to establishing trust in financial operations. This service seamlessly integrates off-chain and on-chain processes, providing a robust framework to verify and authenticate reserve holdings.
+
+
+A Proof of Reserve is defined by a pair of an adapter and an aggregator, and can be accessed on-chain through a `POR Proxy` smart contract. The `POR Proxy` is an auxiliary contract that redirects read requests to `POR` contract. The `POR` contract holds all submission values that is served to consumers through `POR Proxy` contract.
```

Comment:
Let's say that the on-chain implementation is same as for Data Feed. And the we will reference to real names of smart contracts `Aggregator` and `AggregatorProxy`. 🙌